contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -95,19 +95,24 @@ namespace eosiosystem {
    using blockchain_parameters_t = eosio::blockchain_parameters;
 #endif
 
```

Comment:
Remove include of `unordered_set`.

Reply:
Removed it. https://github.com/eosnetworkfoundation/eos-system-contracts/pull/189/commits/961349b90018390aaa1fc8b7f13668d6a3a7880d